records/test2.yaml

```diff
@@ -0,0 +1,24 @@
+Collection Method: 
+  - Laboratory
+Country of Origin:
+  - International
+Developer: Testing
+Discipline:
+  - Materials
+Funding: WPTO
+License Type: 
+  - Open-Source
+Lifecycle: 
```

Comment:
`Lifecycle` is invalid in the current schema. This should be `Life Cycle`.

Reply:
This was how it was spelled in the MRE database schema, following [convention](https://en.wikipedia.org/wiki/Life_cycle).

Reply:
Thanks, I'll change it here, but it needs to be updated to lifecyle.